src/Tests/Utils/Helpers.test.ts

```diff
@@ -49,6 +38,31 @@ suite('Utils', function() {
         let realPath = RealPath.createRealPath('/dummy/not/exists/here');
         assert.isNull(realPath);
       });
+
+      test('NEG: cannot create when rawpath includes invalud path - 1', function() {
```

Comment:
```suggestion
      test('NEG: cannot create when rawpath includes invalid path - 1', function() {
```

following tests has typo too. :wink: